src/ipc/ipc4/dai.c

```diff
@@ -346,9 +346,9 @@ int dai_config(struct comp_dev *dev, struct ipc_config_dai *common_config,
 }
 
 #if CONFIG_ZEPHYR_NATIVE_DRIVERS
-int dai_position(struct comp_dev *dev, struct sof_ipc_stream_posn *posn)
+static int dai_zephyr_position(struct dai_data *dd, struct comp_dev *dev,
+			       struct sof_ipc_stream_posn *posn)
```

Comment:
it should, there is nothing changed, just go inside function and replace with platform_dai_wallclock()

Reply:
@ranj063 I meant, that `platform_dai_wallclock()` takes a component pointer as an argument, and after this series the DAI component is gone, so it will be called with a different - copier component? In a couple of `platform_dai_wallclock()` implementations that argument is dutifully ignored, but... In fact maybe the `dev` pointer can be removed there - it's a **wall** clock so it shouldn't need a component context?